blocks/library/gallery/gallery-image.js

```diff
@@ -36,26 +36,28 @@ class GalleryImage extends Component {
 
 		const img = url ? <img src={ url } alt={ alt } data-id={ id } /> : <Spinner />;
 
-		const className = classnames( 'blocks-gallery-image', {
+		const className = classnames( 'blocks-gallery-item', {
 			'is-selected': isSelected,
 		} );
 
 		// Disable reason: Each block can be selected by clicking on it and we should keep the same saved markup
 		/* eslint-disable jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/onclick-has-role, jsx-a11y/click-events-have-key-events */
 		return (
-			<figure className={ className } onClick={ onClick }>
-				{ isSelected &&
-					<div className="blocks-gallery-image__inline-menu">
-						<IconButton
-							icon="no-alt"
-							onClick={ onRemove }
-							className="blocks-gallery-image__remove"
-							label={ __( 'Remove Image' ) }
-						/>
-					</div>
-				}
-				{ href ? <a href={ href }>{ img }</a> : img }
-			</figure>
+			<li className={ className } onClick={ onClick }>
```

Comment:
Sounds a bit weird to use an `li` as the root element of a component since an `li` requires its parent to be `ul` and this can't be enforced here, I wonder if we can move the `li` outside the `GalleryImage` component